libs/unity/library/Runtime/Scripts/PeerConnection.cs

```diff
@@ -135,15 +135,6 @@ public class PeerConnection : WorkQueue
 
         #region Behavior settings
 
-        /// <summary>
-        /// Initialize the peer connection on <a href="https://docs.unity3d.com/ScriptReference/MonoBehaviour.Start.html">MonoBehaviour.Start()</a>.
-        /// If this field is <c>false</c> then the user needs to call <see cref="InitializeAsync(CancellationToken)"/>
-        /// to manually initialize the peer connection before it can be used for any purpose.
-        /// </summary>
-        [Tooltip("Automatically initialize the peer connection on Start().")]
-        [Editor.ToggleLeft]
-        public bool AutoInitializeOnStart = true;
```

Comment:
We should probably document in the class header this new behavior.

Reply:
Fixed